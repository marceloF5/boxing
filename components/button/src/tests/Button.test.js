import React from 'react';
import { mount } from 'enzyme';
import Button from '../Button';
import 'jest-styled-components';

// const mockForSnapshot = displayName => {
//     const component = function Component() {
//         return null;
//     };
//     component.displayName = displayName;
//     return component;
// };

describe('<Button />', () => {
    test('should render primary button', () => {
        const wrapper = mount(<Button>Add to Bag</Button>);
        const button = wrapper.find('button').debug();

        expect(button).toMatchSnapshot();
    });

    test('should render secondary button', () => {
        const wrapper = mount(<Button variant="secondary">Add to Bag</Button>);
        const button = wrapper.find('button').debug();

        expect(button).toMatchSnapshot();
    });
});
