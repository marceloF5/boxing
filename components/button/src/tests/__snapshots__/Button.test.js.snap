// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should render primary button 1`] = `
"<button disabled={false} className=\\"Button__StyledButton-pqwbd2-0 lhIwQt\\">
  Add to Bag
</button>"
`;

exports[`<Button /> should render secondary button 1`] = `
"<button disabled={false} className=\\"Button__StyledButton-pqwbd2-0 leOBmX\\">
  Add to Bag
</button>"
`;
